WeWatch/WeWatch/Splash/SplashView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SplashViewModel.swift
+//  WeWatch
+//
+//  Created by Anton on 18/12/2024.
+//
+
+import SwiftUI
+
+struct SplashView: View {
+    
+    @StateObject var viewModel = SplashViewModel()
+    
+    var body: some View {
+        if viewModel.showMainView {
+            ContentView()
+        } else {
+            ZStack{
+                Color(.black)
+                    .ignoresSafeArea()
+                VStack{
+                    HStack(spacing: 0) {
+                        Text("We")
+                            .foregroundColor(.whiteColor)
```

Comment:
Split entire `ZStack` to separate property, like: 

`private var splashViewContent: some View { `